packages/salesforcedx-utils-vscode/test/jest/helpers/paths.test.ts

```diff
@@ -70,4 +73,44 @@ describe('test project paths', () => {
       expect(relativeStateFolder).toEqual(FAKE_STATE_FOLDER);
     });
   });
+
+  describe('test SFDXGlobalConfigPath', () => {
+    it('should return the global config path', () => {
+      const mockOS = jest.requireActual('os');
+      mockOS.homedir = jest.fn().mockReturnValue(FAKE_HOME_DIR);
+      jest.mock('os', () => mockOS);
```

Comment:
Nice job figuring this out.  I wonder, can the` import `statement at the top be removed with the use of `requireActual` here?

Reply:
Yes! Thank you :)